src/CommunityToolkit.Maui.Camera/CameraManager.macios.cs

```diff
@@ -0,0 +1,299 @@
+﻿using System.Diagnostics;
+using System.Runtime.InteropServices;
+using AVFoundation;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Extensions;
+using CoreMedia;
+using Foundation;
+using UIKit;
+using static UIKit.UIGestureRecognizer;
+
+namespace CommunityToolkit.Maui.Core;
+
+partial class CameraManager
+{
+	// TODO: Check if we really need this
+	readonly NSDictionary<NSString, NSObject> codecSettings = new([AVVideo.CodecKey], [new NSString("jpeg")]);
+
+	AVCaptureSession? captureSession;
+	AVCapturePhotoOutput? photoOutput;
+	AVCaptureInput? captureInput;
+	AVCaptureDevice? captureDevice;
+
+	AVCaptureFlashMode flashMode;
+
+	// IN the future change the return type to be an alias
+	public UIView CreatePlatformView()
+	{
+		captureSession = new AVCaptureSession
+		{
+			SessionPreset = AVCaptureSession.PresetPhoto
+		};
+
+		var previewView = new PreviewView();
+		previewView.Session = captureSession;
+
+		return previewView;
+	}
+
+	/// <inheritdoc/>
+	public void Dispose()
+	{
+		Dispose(true);
+		GC.SuppressFinalize(this);
+	}
+
+	public partial void UpdateFlashMode(CameraFlashMode flashMode)
+	{
+		this.flashMode = flashMode.ToPlatform();
+	}
+
+	public partial void UpdateZoom(float zoomLevel)
+	{
+		if (!IsInitialized || captureDevice is null)
+		{
+			return;
+		}
+
+		if (zoomLevel < (float)captureDevice.MinAvailableVideoZoomFactor || zoomLevel > (float)captureDevice.MaxAvailableVideoZoomFactor)
+		{
+			return;
+		}
+
+		captureDevice.LockForConfiguration(out NSError error);
+		if (error is not null)
+		{
+			Console.WriteLine(error);
+			Debug.WriteLine(error);
```

Comment:
Just `Trace` is presented in `Release` mode
```suggestion
			System.Diagnostics.Trace.WriteLine(error);
```